modules/virt-disable-individual-bootsource-update.adoc

```diff
@@ -5,13 +5,16 @@
 
 :_content-type: PROCEDURE
 [id="virt-disable-individual-bootsource-update_{context}"]
-= Disabling an automatic update for a custom boot source
+= Disabling an automatic update for a single system-defined or user-defined boot source
 
-You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for an individual custom boot source.
+System-defined boot sources are not listed by default in the `spec.dataImportCronTemplates` of the `HyperConverged` custom resource (CR).
+
+You can disable automatic imports and updates for a single user-defined boot source and for a single system-defined boot source.
 
 .Procedure
 
-* Edit the `HyperConverged` custom resource (CR) and disable updates for a custom boot source by setting the `dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.
+* To disable automatic imports and updates for a single user-defined boot source, remove the boot source from the `spec.dataImportCronTemplates` field in the custom resource list.
+* To disable automatic imports and updates for a single system-defined boot source, edit the `HyperConverged` CR and disable automatic imports and updates, setting the `spec.dataImportCronTemplates.dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.
```

Comment:
`spec.dataImportCronTemplates.dataimportcrontemplate.kubevirt.io/enable` this should be updated. To disable a system-defined boot source, we need to update hco.spec.dataImportCronTemplates; add an entry for the system-defined boot source to be disabled there; and for that entry we need to add the metadata.annotation: `dataimportcrontemplate.kubevirt.io/enable: 'false'`
Here is an example: https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/docs/cluster-configuration.md#disabling-a-common-golden-image

Reply:
Thanks!